R/runDrugResponseProcessingPipeline.R

```diff
@@ -234,8 +234,7 @@ runDrugResponseProcessingPipeline <- function(x,
   checkmate::assert_choice(start_from, get_pipeline_steps())
   checkmate::assert_character(selected_experiments, null.ok = TRUE)
   checkmate::assert_subset(selected_experiments,
-                           names(gDRutils::get_experiment_groups()),
-  )
+                           gDRutils::get_supported_experiments())
```

Comment:
@bczech Please, confirm that this change won't cause any issue.
previous
```
> names(gDRutils::get_experiment_groups())
[1] "single-agent" "combination" 
```
current
```
> gDRutils::get_supported_experiments()
[1] "single-agent" "combination"  "co-dilution" 
```

Reply:
it's ok

Reply:
✔️